code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -978,3 +978,8 @@
 	var/datum/computer_file/program/robotact/program = modularInterface.get_robotact()
 	if(program)
 		program.force_full_update()
+
+/mob/living/silicon/robot/get_exp_list(minutes)
+	. = ..()
+
+	.[EXP_TYPE_JOB_CYBORG] = minutes
```

Comment:
```suggestion
/mob/living/silicon/robot/get_exp_list(minutes)
	. = ..()
	var/static/datum/job/cyborg/robot_job = /datum/job/cyborg
	var/static/robot_job_name = initial(robot_job.name)
	.[robot_job_name] = minutes
```

Reply:
Handled through SSjob and grabbing a static reference to a job datum singleton we already have kicking around.